packages/icons/update-icons.js

```diff
@@ -39,7 +39,9 @@ const list = `${generatedWarning}export const IconsList = [\n\t${icons
 
 writeFileSync(join(__dirname, './icons-list.ts'), list);
 
-const scssConfig = `${generatedWarning}$font-path: '//cdn.lucca.fr/lucca-front/icons/font/lucca-icons' !default;
+const scssConfig = `${generatedWarning}// to test locally (without the CDN)
+// $font-path: '../../font/lucca-icons' !default;
```

Comment:
normal de garder en commentaire cette ligne ?

Reply:
Yes, it's not easy to recover this path.